axelrod/tests/strategies/test_calculator.py

```diff
@@ -35,16 +36,65 @@ def test_strategy(self):
         P2.history = history
         self.assertEqual(C, P1.strategy(P2))
 
-        # Test post 20 rounds responses
-        self.responses_test([D], [C] * 21, [C] * 21)
-        history = [C, C, D, C, C, D, C, C, C, D, C, C, C, C, D, C, C, C, C, C,
-                   D]
-        self.responses_test([D], [C] * 21, history)
-        history = [C, C, D, C, C, D, C, C, C, D, C, C, C, C, D, C, C, C, C, C,
-                   D, C]
-        self.responses_test([C], [C] * 22, history)
+    def test_twenty_rounds_joss_then_defects_for_cyclers(self):
+        """uses axelrod.strategies.axelrod_first.Joss strategy for first 20 rounds"""
+        seed = 2
+        seed_two_flip_indices = [1, 3]
+        twenty_alternator_actions = [C, D] * 10
+        twenty_test_actions = get_joss_strategy_actions(twenty_alternator_actions, seed_two_flip_indices)
+
+        expected_actions = twenty_test_actions + [(D, C), (D, D), (D, C), (D, D)]
+        self.versus_test(axelrod.Alternator(), twenty_test_actions, seed=seed)
+        self.versus_test(axelrod.Alternator(), expected_actions, seed=seed)
+
+    def test_twenty_rounds_joss_then_tit_for_tat_for_non_cyclers(self):
+        """uses axelrod.strategies.axelrod_first.Joss strategy for first 20 rounds"""
+        seed = 2
+        seed_two_flip_indices = [1, 2]
```

Comment:
could you just rename this to `indices_to_flip` (which matches the function var).

No need for the seed detail.